### ISSUE TITLE
openstack builder failing with 1.3.0

### ISSUE THREAD

**Issue by furlongm** [hashicorp/packer#6709]
openstack builder doesn't seem to work with 1.3.0:

```
openstack output will be in this color.

==> openstack: Loading flavor: generic2.small
    openstack: Verified flavor. ID: 357a88f8-2af7-414c-a197-a2db9bda4e7b
==> openstack: Pausing after run of step 'StepLoadFlavor'. Press enter to continue. 
==> openstack: Creating temporary keypair: packer_5b9ad626-cd24-d24c-a3bc-b5724a7523c1 ...
==> openstack: Created temporary keypair: packer_5b9ad626-cd24-d24c-a3bc-b5724a7523c1
==> openstack: Successfully converted BER encoded SSH key to DER encoding.
    openstack: Saving key for debug purposes: os_openstack.pem
==> openstack: Pausing after run of step 'StepKeyPair'. Press enter to continue. 
==> openstack: Pausing after run of step 'StepSourceImageInfo'. Press enter to continue. 
==> openstack: Pausing after run of step 'StepCreateVolume'. Press enter to continue. 
==> openstack: Launching server...
==> openstack: Launching server...
    openstack: Server ID: 608f6674-d8a8-4355-9e28-174c1e5c46e6
==> openstack: Waiting for server to become ready...
==> openstack: Pausing after run of step 'StepRunSourceServer'. Press enter to continue. 
==> openstack: Pausing after run of step 'StepGetPassword'. Press enter to continue. 
==> openstack: Pausing after run of step 'StepWaitForRackConnect'. Press enter to continue. 
==> openstack: Pausing before cleanup of step 'StepWaitForRackConnect'. Press enter to continue. 
==> openstack: Pausing before cleanup of step 'StepGetPassword'. Press enter to continue. 
==> openstack: Pausing before cleanup of step 'StepRunSourceServer'. Press enter to continue. 
==> openstack: Terminating the source server: 608f6674-d8a8-4355-9e28-174c1e5c46e6 ...
==> openstack: Pausing before cleanup of step 'StepCreateVolume'. Press enter to continue. 
==> openstack: Pausing before cleanup of step 'StepSourceImageInfo'. Press enter to continue. 
==> openstack: Pausing before cleanup of step 'StepKeyPair'. Press enter to continue. 
==> openstack: Deleting temporary keypair: packer_5b9ad626-cd24-d24c-a3bc-b5724a7523c1 ...
==> openstack: Pausing before cleanup of step 'StepLoadFlavor'. Press enter to continue. 
Build 'openstack' errored: unexpected EOF

==> Some builds didn't complete successfully and had errors:
--> openstack: unexpected EOF
```

**Comment by furlongm** [comment]
1.2.5 works as expected.

**Comment by SwampDragons** [comment]
Can you share steps to reproduce (e.g. the simplest possible packer config we'd need to see this error ourselves?)

**Comment by azr** [comment]
Howdy, I believe this was fixed by #6739, StepLoadFlavor is run right before StepKeyPair which I expected to panic when I made that PR.

So, I'm closing this issue; please comment/reopen it if it's not solved 🙂
